test/browser/select.test.js

```diff
@@ -46,6 +46,21 @@ describe('Select', () => {
 		expect(scratch.firstChild.value).to.equal('B');
 	});
 
+	it('should alias onInput to onChange', () => {
+		const func = () => {};
+		render(
+			<select onInput={func}>
+				<option value="A">A</option>
+				<option value="B">B</option>
+				<option value="C">C</option>
+			</select>,
+			scratch
+		);
+
+		expect(scratch.firstChild._listeners.change).to.not.be.undefined;
```

Comment:
I think to get this test to pass, don't rely on the internal properties. Instead, use our `createEvent` helper to fire a `change` event and verify the `onInput` spy is called. See tests in `events.test.js` for an example